src/com/pentalog/pentastagiu/homework2/shop/ItemsOnSale.java

```diff
@@ -0,0 +1,11 @@
+package com.pentalog.pentastagiu.homework2.shop;
+//
+public class ItemsOnSale {
+
+    double price;
+
+
+    ItemsOnSale(double price){
+        this.price=price;
```

Comment:
ar trebui sa adaugi o validare pe pret si cantitate pentru a nu permite valori negative, care ar duce la crearea unor obiecte invalide dpdv business; daca arunci o exceptie din constructor in astfel de cazuri, impiedici astfel crearea de obiecte invalide